src/XMakeBuildEngine/BackEnd/Components/Logging/EventSourceSink.cs

```diff
@@ -129,6 +130,24 @@ public bool HaveLoggedBuildFinishedEvent
             get;
             set;
         }
+
+        /// <summary>
+        /// A list of warnings to treat as errors.  If null, nothing is treated as an error.  If an empty set, all warnings are treated as errors.
```

Comment:
This set is the warning codes to treat as errors.  When the `EventSourceSink` is created, this property is set to null, an empty collection, or a collection with items.  There is no magic number for this scenario.